README.md

```diff
@@ -465,6 +465,22 @@ this way, then 'none' will be used. Additionally, violation reporting is throttl
 spamming the telemetry service with repetitive data. Identical violations will not be
 reported more than once a day.
 
+In case of local developement of the dynamic plugin, just pass needed CSP directives address to the console server, using the `--content-security-policy` flag.
+
+Example:
+
+```
+./bin/bridge --content-security-policy ScriptSrc="localhost:1234 localhost:2345" --content-security-policy FontSrc="localhost:3456 localhost:4567"
```

Comment:
Here I would probably the directive name `font-src` instead of `FontSrc`

```suggestion
./bin/bridge --content-security-policy ScriptSrc="localhost:1234 localhost:2345" --content-security-policy font-src="localhost:3456 localhost:4567"
```

Reply:
I was thinking about it, but then we would need to change the `console-config.yaml` representation, or accept both formats `FontSrc` & `font-src`. Here I would rather go with the first option.

Reply:
I don't know if we need to change console-config.yaml. Can we just translate between the names in config.go instead?

I wouldn't accept both formats. Let's pick one and go with it. If it's difficult, we can stick with PascalCase.

Reply:
We will need the change to the console-config.yaml representation. Currently the in console-config/yaml CSPs are set in `FontSrc="localhost:3456 localhost:4567"` format. Thats how they are passed to the `--content-security-policy` flag by bridge.  OR we need to update the `addContentSecurityPolicy()` in `pkg/serverconfig/config.go` so the `cspDirectiveName` is transformed into the `font-src` format.